.github/actions/helm-default/action.yaml

```diff
@@ -21,11 +21,7 @@ runs:
     - id: set-defaults
       shell: bash
       run: |
-        if [ "${{ github.event_name }}" = "workflow_dispatch" ] || [ ${{ github.event.pull_request }} ] ; then
-          SHA="${{ inputs.image-tag }}"
-        else
-          SHA="${{ github.sha }}"
-        fi
+        SHA="${{ inputs.image-tag }}"
```

Comment:
We should probably remove `if [ -n "${SHA}" ]; then` below in this case if we expect caller to always specify correct SHA?

Reply:
Makes sense, updated by the second commit. I didn't modify that part previously because the tag was already always specified in the previous implementation (either via `image-tag` or due to defaulting), and the other commit wasn't changing anything in that respect. However, I agree that it still makes sense to simplify that logic.